Source/Menu/MainForm.cs

```diff
@@ -264,6 +262,31 @@ orderby tool.Text
             }
         }
 
+        private void LoadDocuments(List<ToolStripItem> docs, string folderPath, string code)
+        {
+            foreach (var filePath in Directory.GetFiles(folderPath))
+            {
+                var ext = System.IO.Path.GetExtension(filePath);
+                var name = System.IO.Path.GetFileNameWithoutExtension(filePath);
+                // These are the following formats that can be selected.
+                if (".pdf .doc .docx .pptx .txt".Contains(ext))
+                {
+                    var codeLabel = "";
+                    if (code != null)
+                        codeLabel = $" [{code}]";
```

Comment:
```suggestion
                    var codeLabel = string.IsNullOrEmpty(code) ? "" : $" [{code}]";
```

Reply:
3 lines collapse to 1 but is it easier to read?